src/index.html

```diff
@@ -7,8 +7,26 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home"><img src="images/logo.png" alt="logo"></a>
+      <div class="spacer"></div>
+      <nav>
+        <ul>
+          <li><a href="#" class="is-active"><span>Apple</span></a></li>
```

Comment:
Please refer to this [guideline](https://mate-academy.github.io/style-guides/htmlcss.html#html-line-wrapping) and fix the formatting

Reply:
Whenever I'm trying to apply HTML line-wrapping then I've got a lot of errors provided by linter so I guess I have to just leave it as it is.